spec/lucky/file_response_spec.cr

```diff
@@ -0,0 +1,101 @@
+require "../spec_helper"
+
+include ContextHelper
+
+describe Lucky::FileResponse do
+  describe "#print" do
+    describe "file is missing" do
+      it "raises an exception" do
+        context = build_context
+
+        expect_raises Lucky::Exceptions::MissingFile, /^Cannot read file/ do
+          print_file_response(context, "nope")
```

Comment:
One cool feature in Crystal is that you can use keyword args for any argument (you don't need to specify them in the method signature as a keyword arg). So you could do `print_file_response(context, file: "nope")` so it is clearer what the argument is for. This could be done in the other ones as well